.github/workflows/build.yml

```
@@ -0,0 +1,38 @@
+name: Update the build branch
+
+on:
+  push:
+    branches: [ 'trunk' ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build and push to branch
+    steps:
+      - name: git-checkout
+        uses: actions/checkout@v2
+
+      - name: Install all dependencies
+        run: yarn run initial-setup
+
+      - name: Build
+        run: yarn workspaces run build
+
+      - name: Remove Source files
+        run: |
+          rm -rf source/wp-content/plugins/wporg-internal-notes/src
+          rm -rf source/wp-content/plugins/wporg-internal-notes/node_modules
+
+      - name: Create package.json
+        run: |
+          cd source/wp-content/plugins/wporg-internal-notes/
+          cat <<<$( jq '{ name, version, description, author, homepage, license, support }' package.json ) > package.json
```

```suggestion
      - name: Create composer.json
        run: |
          cat <<<$( jq '{ name, description, homepage, license, support }' composer.json ) > source/wp-content/plugins/wporg-internal-notes/composer.json
```

Er, as mentioned [here](https://github.com/WordPress/wporg-internal-notes/pull/8/files#r815193208), I think it'd be good to add a composer.json file, but I don't think keeping the package.json is necessary...

---

Oh 🤦  package.json vs composer.json, silly me..